File: src/commands/utility/info.js
const util = require("../../util");
const Discord = require("discord.js");

module.exports = {
    name: "info",

    aliases: ["information", "support", "invite"],

    description: "Information about Pingernos",

    async execute(message, args, client) {
        const embed = new Discord.MessageEmbed()
            .setDescription(
                `${client.user.username} is not affiliated with Aternos GmbH, but is a bot developed by \
                 [PsychOps Studios](https://psychops.tk) with the purpose of offering Aternos users a public alternative to the \
                  private Aternos bot.\n\n[[Invite]](${util.links.invite}) [[Support]](${util.links.support}) \
                   [[Github]](https://github.com/PsychOps/serverpinger) [[Aternos]](https://aternos.org/)`
            )
            .setColor(util.color.blue);

        await message.reply({ embeds: [embed] });
    },
};

File: cogs/info.py
from asyncio import wait_for

import discord
from discord import slash_command, Embed

from utilities.data import Colors, get_server_status


class Info(discord.Cog):
    def __init__(self, bot: discord.Bot):
        self.bot = bot

    @slash_command()
    async def info(self, ctx):
        """ Displays information about Pingernos """
        embed = Embed()
        try:
            stat = await wait_for(get_server_status("example.aternos.me"), timeout=2)
        except TimeoutError:
            latency = "N/A"
        else:
            latency = round(stat.latency)
        embed.description = f"""
{self.bot.user.name} is a bot developed by Blue Atomic to service Aternos users by displaying server statuses. We are not affiliated with Aternos and are an independent FOSSDEV organization.

**Guilds:** {len(self.bot.guilds)}
**Users:** {sum(x.member_count for x in self.bot.guilds)}
**API Latency:** {round(self.bot.latency * 1000)}ms
**Protocol Latency:** {latency}ms
**Shard count:** {self.bot.shard_count}
**Shard:** {ctx.guild.shard_id + 1}

[[Invite]](https://discord.com/api/oauth2/authorize?client_id=889197952994791434&permissions=274878286912&scope=bot%20applications.commands) [[Support]](https://discord.gg/Ukr89GrMBk) [[Github]](https://github.com/BlackFurORG/pingernos) [[Privacy Policy]](https://gist.github.com/MiataBoy/20fda9024f277ea5eb2421adbebc2f23) [[Terms of Service]](https://gist.github.com/MiataBoy/81e96023a2aa055a038edab02e7e7792)
        """
        embed.colour = Colors.blue
        await ctx.respond(embed=embed)


def setup(bot: discord.Bot):
    bot.add_cog(Info(bot))
